packages/documentation/copy/en/developer/Regression Tests.md

```diff
@@ -6,110 +6,66 @@ oneline: "Regression Tests for Lingua Franca."
+- **Integration tests** are complete Lingua Franca programs that are compiled and executed automatically. A test passes if it successfully compiles and runs to completion with normal termination (return code 0). These tests are located in the `test` directory at the root of the LF repo, with one subdirectory per target language.
+Their implementation can be found in the `core/src/integrationTest` directory.
+The integration tests are also executed through JUnit using methods with `@Test` directives, but they are executed separately.
 
-- **System tests** are complete Lingua Franca programs that are compiled and executed automatically. A test passes if it successfully compiles and runs to completion with normal termination (return code 0). These tests are located in the `test` directory at the root of the LF repo, with one subdirectory per target language.
-- **Unit tests** are Java methods in the [`compiler` package](https://github.com/lf-lang/lingua-franca/tree/master/org.lflang.tests/src/org/lflang/tests/compiler) that are labeled with the `@test` directive in the comment just before the method. These tests check individual functions of the code generation infrastructure and/or IDE. The system tests are also executed through JUnit using methods with `@test` directives, but these tests are located in the [`runtime` package](https://github.com/lf-lang/lingua-franca/tree/master/org.lflang.tests/src/org/lflang/tests/runtime) (because their goal is not only to successfully generate code, but also to execute it and test for expected results).
-- **Examples** are complete Lingua Franca programs that are compiled but not executed automatically. These are not executed automatically because they are typically interactive and may not terminate.
+## Running the Tests From the Command Line
 
-To learn about Lingua Franca, browsing the system tests and examples could be very useful.
+To run all unit tests, simply run `./gradlew test`. Note that also the normal build tasks `./gradlew build` runs all the unit tests.
 
-### Browsing and Editing Examples in Epoch
 
-- In the LF IDE Eclipse, select File->New->Project (a General Project is adequate) and click Next.
-- Give the project a name, e.g. `CExamples`.
-- Unselect "Use default location".
-- Navigate to the example directory and click Open. For the C target, the example directory is `$LF/example/C` where `$LF` is the root of your git clone of the lingua-franca repo.
-- Click Finish.
-- Open any one of the `.lf` files in any of the subdirectories.
-- **IMPORTANT:** A dialog appears: Do you want to convert 'test' to an Xtext Project? Say YES.
+The integration tests can be run using the `integratioTest` task. However, typically it is not desired to run all tests for all targets locally as it will need the right target tooling and will take a long time.
```

Comment:
Should this be `integrationTest`?

Reply:
```suggestion
The integration tests can be run using the `integrationTest` task. However, typically it is not desired to run all tests for all targets locally as it will need the right target tooling and will take a long time.
```